packages/db/index.js

```diff
@@ -42,7 +42,7 @@ const userQueryWithTags = `
   publicAddress:public_address,
   created:inserted_at,
   updated:updated_at,
-  tags:user_tag_user_id_fkey(user_id,id,tag,value)
+  tags:user_tag_user_id_fkey(user_id,id,tag,value, deleted_at)
```

Comment:
thanks for fixing this.  looks like I lost this field when resolving the merge conflicts from all the template strings being extracted into consts.

Reply:
👍 no worries.

Reply:
@adamalton how does this work?  _why_ does adding `deleted_at` here change anything?  `hasTag` checks for `!tag.deleted_at` before confirming the value.

Reply:
@adamalton so i think i get it.  because `public.user` _does not contain_ a `deleted_at` column, the matching algorithm will require the tag's `deleted_at` value to be null?

Reply:
@jsdevel without this, the value of the `deleted_at` column isn't returned by the query, and so when `hasTag` does its filtering `tag.deleted_at` will always evaluate as false-y because `tag.deleted_at` is `undefined`.

Reply:
if that is the case, then this doesn't really "fix" anything because the code is currently looking for `!tag.deleted_at` anyway...

Reply:
ahh, makes sens.  thank you!